### PR TITLE
commented line 26

### PR DESCRIPTION
**Pull request by jeelion22**
review

### CODE REVIEW

File: hangman.py [hunk 1]
```diff
@@ -23,7 +23,7 @@
 
 """
 
-stored_letters = list("java")  # converts string to list
+# stored_letters = list("java")  # converts string to list
 guessed_word = len(stored_letters) * ["_"]
 used_letters = list()
```
Comment (josef-o) [inline hangman.py:29]: count_balance is going down so just with this it becomes an infinite loop.

I know there is a break inside the loop but having both conditions is confusing as to which one matters.

Comment (josef-o) [inline hangman.py:29]: A way to do this in one line is using the same case on both sides:
```
stored_letters[i].upper() == guess.upper()
```

**Review by josef-o — CHANGES_REQUESTED** [review]
Changes suggested

File: hangman.py (hunk 1 above)
Comment (jeelion22) [reply hangman.py]: I changed the confusing condition in the while loop. the committed file is in my development branch

Comment (jeelion22) [reply hangman.py]: changes are done

**Comment by josef-o** [comment]
The changes are good!

One thing @jeelion22 is that closing the PR means that it is rejected and the requested changes won't be merged to the `main` branch.

**Pull request closed without merging**